.github/workflows/linux-build.yml

```
@@ -44,15 +40,132 @@ on:
+          ccache -vs
+      - name: Install Docs Dependencies
+        run: |
+          sudo apt install -y --no-install-recommends pandoc
+          sudo apt install -y pipx
+          pipx install --force uv cmake==3.31
+          echo $GITHUB_ENV
+          echo "PATH=$PATH:/github/home/.local/bin" >> $GITHUB_ENV
+          which uv
+          make python-venv
+          which uv
+          uv pip install -r scripts/docs-requirements.txt 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)